src/adstaylorinteg.jl

```diff
@@ -0,0 +1,326 @@
+# This file is part of the TaylorIntegration.jl package; MIT licensed
+
+using LsqFit: curve_fit
+using AbstractTrees: Leaves, getroot
+
+# Exponential model y(t) = A * exp(B * t)
+# used to estimate the truncation error
+# of jet transport polynomials
+exp_model(t, p) = p[1] * exp(p[2] * t)
+exp_model!(F, t, p) = (@. F = p[1] * exp(p[2] * t))
+
+# In place jacobian of exp_model! wrt parameters p
+function exp_model_jacobian!(J::Array{T, 2}, t, p) where {T <: Real}
+    @. J[:, 1] = exp(p[2] * t)
+    @. @views J[:, 2] = t * p[1] * J[:, 1]
+end
+
+"""
+    size_per_variable(P::TaylorN{T}) where {T <: Real}
+    size_per_variable(P::AbstractVector{TaylorN{T}}) where {T <: Real}
+
+Return the truncation error of each jet transport variable in `P`.
+
+!!! reference
+    See section 3 of https://doi.org/10.1007/s10569-015-9618-3.
+"""
+function size_per_variable(P::TaylorN{T}) where {T <: Real}
```

Comment:
Given that it refers to truncation error, I'd suggest to rename `size_per_variable` to something like `trunc_error`, or `error_per_variable`?

Reply:
I'll rename this function to `truncerror`.